website/docs/reference/integrations/slack-app.md

```diff
@@ -0,0 +1,64 @@
+---
+id: slack-app
+title: Slack App
+---
+
+:::info Availability
+
+The Slack App integration was introduced in **Unleash 5.5**.
+
+:::
+
+The Slack App integration posts messages to the selected channels in your Slack workspace.
+
+## Installation {#installation}
+
+To install the Slack App integration, follow these steps:
+
+1. Navigate to the *integrations* page in the Unleash admin UI (available at the URL `/integrations`) and click "configure" on the Slack App integration.
+2. On the integration configuration form, click the "install & connect" button.
```

Comment:
small nit: click is something you only do with a mouse, generally. With touch and keyboard, you don't really 'click', so I suggest using words that don't rely on the medium.

```suggestion
1. Navigate to the *integrations* page in the Unleash admin UI (available at the URL `/integrations`) and select "configure" on the Slack App integration.
2. On the integration configuration form, use the "install & connect" button.
```

Reply:
Oh right, good point!